sim/shaman/spirit_wolves.go

```diff
@@ -0,0 +1,116 @@
+package shaman
+
+import (
+	"math"
+	"strconv"
+	"time"
+
+	"github.com/wowsims/wotlk/sim/core"
+	"github.com/wowsims/wotlk/sim/core/proto"
+	"github.com/wowsims/wotlk/sim/core/stats"
+)
+
+type SpiritWolf struct {
+	core.Pet
+
+	shamanOwner *Shaman
+}
+
+type SpiritWolves struct {
+	SpiritWolf1 *SpiritWolf
+	SpiritWolf2 *SpiritWolf
+}
+
+func (SpiritWolves *SpiritWolves) EnableWithTimeout(sim *core.Simulation) {
+	SpiritWolves.SpiritWolf1.EnableWithTimeout(sim, SpiritWolves.SpiritWolf1, time.Second*45)
+	SpiritWolves.SpiritWolf2.EnableWithTimeout(sim, SpiritWolves.SpiritWolf1, time.Second*45)
+}
+
+func (SpiritWolves *SpiritWolves) CancelGCDTimer(sim *core.Simulation) {
+	SpiritWolves.SpiritWolf1.CancelGCDTimer(sim)
+	SpiritWolves.SpiritWolf2.CancelGCDTimer(sim)
+}
+
+// Source: https://web.archive.org/web/20201120214816/https://github.com/dalaranwow/dalaran-wow/issues/4670
+var spiritWolfBaseStats = stats.Stats{
+	stats.Agility:     113,
+	stats.Strength:    331,
+	stats.AttackPower: 836, // uncertain number. 1 STR = 2AP, but surely AGI has a contribution?. This is copy-value from dk/ghoul
```

Comment:
yeah if it uses the same scaling as the shaman it should be 1:1 for both str and agi to AP

Reply:
I've double-checked, and the stats are agi 113, str 331, attack power -20.

Spirit Wolf unbuffed will have 331 str * 2 = 662
-20 yielding 642 attack power.

This checks out with testing.